fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm

```diff
@@ -92,13 +94,13 @@
 		return
 	if(owner.pulling == feed_target && owner.grab_state >= GRAB_AGGRESSIVE)
 		if(!IS_BLOODSUCKER(feed_target) && !IS_VASSAL(feed_target) && !IS_MONSTERHUNTER(feed_target))
-			feed_target.Unconscious((5 SECONDS + level_current))
+			feed_target.Unconscious((5 + level_current) SECONDS)
 		if(!feed_target.density)
 			feed_target.Move(owner.loc)
-	if(owner.pulling == feed_target && owner.grab_state >= GRAB_AGGRESSIVE)
```

Comment:
Why're we removing this?

Reply:
look at the line above, else statement has the same effect